README.md

```diff
@@ -45,7 +45,7 @@ To simplify development of the server module of Screenly OSE, we've created a Do
 Assuming you're in the source code repository, simply run:
 
 ```
-$ docker run --rm -ti \
+$ docker run --rm -it \
```

Comment:
This is the same thing :)

Reply:
I'm sorry about that.  I did that just after I added a travis hook to my local fork, since travis has to wait for a push to start running.  I forgot it was there when I made the PR.  Also, when I tried :8081, I found that it worked ok, but I'm not sure if the ```server.py``` background process was killed by travis.  When ```python -m SimpleHTTPServer 8080 &``` fails to start because 8080 is already used, the ```server.py``` background process is killed by travis at the end of the run, but when I tested with :8081 for just the static server, it seems that the static server process was killed, but the ```server.py``` process wasn't killed.  The ```rtmp.py``` process was killed on both attempts.

Thanks for making this project available!  Y'all have already taught me quite a few things that I have not been doing myself.  I will probably start using ```splinter``` in the future.  The last time I attempted to setup browser testing, it was in a nodejs environment, and I wasn't really happy with it.  I like using python instead.

Reply:
Glad you like it and thanks for the contributions!

Reply:
@vpetersson , I should've mentioned that ```python -m SimpleHTTPServer 8080 &``` only works on python2 and ```python -m six.moves.SimpleHTTPServer 8080``` will not work with either python version on my local machine.  I still don't know what the strategy is for the screenly code to be py3 capable, but using an explicit ```python2``` command should work as long as it is available on travis.